deepmd/pt/model/descriptor/hybrid.py

```
@@ -332,7 +345,9 @@ def forward(
                 nl = nl_distinguish_types[
                     :, :, self.nlist_cut_idx[ii].to(atype_ext.device)
                 ]
-            odescriptor, gr, g2, h2, sw = descrpt(coord_ext, atype_ext, nl, mapping)
+            odescriptor, gr, g2, h2, sw = descrpt(
```

Ruff (0.15.12)

[warning] Unpacked variable `g2` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)